verify/BUILD

```diff
@@ -24,6 +26,25 @@ sh_test(
         "//:all-srcs",
         "@io_bazel_rules_go_toolchain//:toolchain",
     ],
+    tags = ["manual"],
+)
+
+sh_test(
+    name = "verify-pylint",
+    srcs = ["verify-pylint.sh"],
+    data = ["//:all-srcs"],
+    tags = ["manual"],
```

Comment:
So that they don't run if you do `bazel test //...`, but only if you do `bazel test //verify:verify-whatever`.